pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -697,21 +697,16 @@ func describePodSummaryInline(rc *kapi.ReplicationController, includeEmpty bool)
 }
 
 func describePodSummary(rc *kapi.ReplicationController, includeEmpty bool) string {
-	actual, requested := rc.Status.Replicas, rc.Spec.Replicas
-	if actual == requested {
-		switch {
-		case actual == 0:
-			if !includeEmpty {
-				return ""
-			}
-			return "0 pods"
-		case actual > 1:
-			return fmt.Sprintf("%d pods", actual)
-		default:
-			return "1 pod"
+	switch rc.Status.Replicas {
+	case 0:
```

Comment:
> This block is losing the "<# pods I have>/<# of pods I want> pods" output.

Hmm... You removed the line in the test, so I thought it was gone.  I'm still getting the "growing to X"?  If so, fix the test, don't just remove the line entirely.